src/evolutionary_keras/optimizers.py

```diff
@@ -83,7 +80,18 @@ def __init__(
         self.n_nodes = 0
         self.n_generations = 1
 
-        super(NGA, self).__init__(*args, **kwargs)
+        super(NGA, self).__init__(name, **kwargs)

```

Comment:
Is there a reason why *args are not passed here? I think it should go in as well...

Reply:
hmm, not really sure. I think the tf optimizers don't take *args inputs anyway, so we might as well get rid of it. Not sure why it is still in the init of NGA itself or why we pass the name though. 

I don't see any harm in passing args though, so for now we might as well.

Reply:
Well, the base Optimizer does. Actually we should pass the lock as well... I'll do that  https://github.com/tensorflow/tensorflow/blob/e3e0ba57815094e58c6df16ef1b68281418bf4a6/tensorflow/python/training/optimizer.py#L313

Reply:
Oh, I was looking a TF 1. TF 2 takes only name, then you are right, *args should be removed.

Reply:
Oh I see, you're right. Sorry about that.